src/current/cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -50,6 +50,10 @@ During enablement of the feature, a list of affected members is shown, and those
 
 ### Ensure that at least one organization admin belongs to no other CockroachDB {{ site.data.products.cloud }} organization
 
+{{site.data.alerts.callout_success}}
+You can now use[Folders]({% link cockroachcloud/folders.md %}) (Limited Access) to  to group, organize, and manage access to clusters in a hierarchy within a single CockroachDB Cloud organization. When compared to managing multiple CockroachDB Cloud organizations, this approach simplifies billing and centralizes cluster administration and observability. To learn more, contact your Cockroach Labs account team.
```

Comment:
```suggestion
You can now use [Folders]({% link cockroachcloud/folders.md %}) (Limited Access)  to group, organize, and manage access to clusters in a hierarchy within a single CockroachDB Cloud organization. When compared to managing multiple CockroachDB Cloud organizations, this approach simplifies billing and centralizes cluster administration and observability. To learn more, contact your Cockroach Labs account team.
```

Reply:
I would also maybe change the second sentence a little, because "this approach" is placed in such a way that it could be referring to "managing multiple" orgs.

What about:
```suggestion
You can now use [Folders]({% link cockroachcloud/folders.md %}) (Limited Access) to group, organize, and manage access to clusters in a hierarchy within a single CockroachDB Cloud organization. In comparison to managing multiple CockroachDB Cloud organizations, Folders simplify billing and centralize cluster administration and observability. To learn more, contact your Cockroach Labs account team.
```

Reply:
(I don't know if we are capitalizing "folders" throughout the docs?)

Reply:
I don't think we do, but the style guide [leaves it up to us](https://github.com/cockroachdb/docs/blob/main/StyleGuide.md#capitalization-rules) and I thought that the word Folders was significant enough in this concept to capitalize it. WDYT?

Reply:
I tweaked your suggestion just a bit before applying. Thanks!